finality-aleph/src/lib.rs

```diff
@@ -54,10 +54,10 @@ pub fn peers_set_config(protocol: Protocol) -> sc_network::config::NonDefaultSet
 
     config.set_config = match protocol {
         Protocol::Validator => sc_network::config::SetConfig {
-            in_peers: 25,
+            in_peers: 0,
             out_peers: 0,
             reserved_nodes: Vec::new(),
-            non_reserved_mode: sc_network::config::NonReservedPeerMode::Accept,
+            non_reserved_mode: sc_network::config::NonReservedPeerMode::Deny,
         },
```

Comment:
would be cool to add a comment here on what it does. At first glance, it looks like we don't allow anyone to connect to us at all.

Reply:
And that is absolutely correct. We only connect based on authentications we get through the other protocol, which we use to make peers reserved.

Reply:
Added comment.